modules/@apostrophecms/schema/ui/apos/logic/AposInputRelationship.js

```diff
@@ -204,13 +206,47 @@ export default {
         : {};
 
       await this.search(qs);
+      if (this.searchList.length) {
+        // psuedo focus first element
+        this.searchFocusIndex = 0;
+      }
     },
     handleFocusOut() {
       // hide search list when click outside the input
       // timeout to execute "@select" method before
       setTimeout(() => {
         this.searchList = [];
       }, 300);
+      this.searchFocusIndex = null;
+    },
+    handleKeydown(event) {
+      switch (event.key) {
+        case 'ArrowDown':
+          if (this.searchFocusIndex + 1 < this.searchList.length) {
+            return this.searchFocusIndex++;
```

Comment:
```suggestion
            this.searchFocusIndex++;
            return;
```
unless there is a specific reason to return an integer here